Anoma/Resource/Types.juvix

```diff
@@ -26,12 +31,15 @@ type NullifierKeyCommitment :=
   mkNullifierKeyCommitment@{unNullifierKeyCommitment : ByteArray};
 
 --- A fixed-size data type encoding a number to be used once ensuring that the resource commitment is unique.
---- NOTE: This should be a number having an at most negligible chance of repeating is sufficient, e.g., a pseudo-random number.
-type Nonce := mkNonce@{unNonce : Nat};
+--- NOTE: This should be a number having an at most negligible chance of repeating, e.g., a pseudo-random number.
+type Nonce := mkNonce@{unNonce : AnomaEncoded};
 
 --- A fixed-size data type encoding a randomness seed.
 --- NOTE: This seed provides pseudo randomness and cannot be expected to provide true randomness.
-type RandSeed := mkRandSeed@{unRandSeed : Nat};
+type RandSeed := mkRandSeed@{unRandSeed : AnomaEncoded};
+
+--- The size of bytes.
+randSeedBytesSize : AnomaEncoded := 32;
```

Comment:
It seems like `randSeedBytesSize` should have type `Nat` (I know they're synonyms but `AnomaEncoded` means the encoded bytes).